### PR TITLE
expression: refactor function.

### PR DESCRIPTION
**Pull request by hanfei1991**
one step of refactor.
@shenli @coocood @zimulala @tiancaiamao PTAL

### CODE REVIEW

File: expression/builtin_info.go [hunk 1]
```diff
@@ -96,16 +88,8 @@ type builtinFoundRowsSig struct {
...
-	args, err := b.evalArgs(row)
-	if err != nil {
-		return types.Datum{}, errors.Trace(err)
-	}
-	return builtinFoundRows(args, b.ctx)
-}
-
-func builtinFoundRows(arg []types.Datum, ctx context.Context) (d types.Datum, err error) {
-	data := ctx.GetSessionVars()
+func (b *builtinFoundRowsSig) eval(_ []types.Datum) (d types.Datum, err error) {
```
Comment (zimulala) [inline expression/builtin_info.go]: Add URL.
Why change the argument from `row` to `_`?

File: expression/builtin_info.go [hunk 2]
```diff
@@ -203,16 +171,8 @@ type builtinConnectionIDSig struct {
...
-	args, err := b.evalArgs(row)
-	if err != nil {
-		return types.Datum{}, errors.Trace(err)
-	}
-	return builtinConnectionID(args, b.ctx)
-}
-
-func builtinConnectionID(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
-	data := ctx.GetSessionVars()
+func (b *builtinConnectionIDSig) eval(row []types.Datum) (d types.Datum, err error) {
```
Comment (zimulala) [inline expression/builtin_info.go]: Why remove `evalArgs` here?

File: expression/builtin_info.go (hunk 1 above)
Comment (hanfei1991) [inline expression/builtin_info.go]: the row always be empty.

File: expression/builtin_info.go (hunk 2 above)
Comment (hanfei1991) [inline expression/builtin_info.go]: the row will always be empty.

**Comment by shenli** [comment]
LGTM

**Comment by hanfei1991** [comment]
@zimulala PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**